classes/Visualizer/Render/Library.php

```diff
@@ -337,6 +337,15 @@ private function _renderChartBox( $placeholder_id, $chart_id, $with_filter = fal
 				admin_url( 'admin-ajax.php' )
 			)
 		);
+		$chart_type = get_post_meta( $chart_id, Visualizer_Plugin::CF_CHART_TYPE, true );
+
+		$types = apply_filters( 'visualizer_pro_chart_types', [] );
+
+		$pro_class = '';
+
+		if ( ! empty( $chart_type ) && isset( $types[ $chart_type ] ) ) {
+			$pro_class = 'is_pro';
+		}
```

Comment:
for the CSS class, I recommend `viz-is-pro-chart.` `is_pro` is not recommended, and `is-pro` might be too generic, and we can have unwanted surprises from other plugins.